metrics/metricstest/metricstest.go

```diff
@@ -118,9 +118,31 @@ func CheckDistributionCount(t test.T, name string, wantTags map[string]string, e
 
 }
 
+// GetLastValueData returns the last value for the given metric, verifying tags.
+func GetLastValueData(t test.T, name string, tags map[string]string) float64 {
+	t.Helper()
+	return GetLastValueDataWithMeter(t, name, tags, nil)
+}
+
+// GetLastValueDataWithMeter returns the last value of the given metric using meter, verifying tags.
+func GetLastValueDataWithMeter(t test.T, name string, tags map[string]string, meter view.Meter) float64 {
```

Comment:
Could we use this func in `CheckLastValueDataWithMeter`? Looks like we can just

```
func CheckLastValueDataWithMeter(t test.T, name string, wantTags map[string]string, wantValue float64, meter view.Meter) {
  t.Helper()
  if GetLastValueDataWithMeter(t, name, tags, meter) != wantValue {
    t.Error("Reporter.Report() wrong value", "metric", name, "got", s.Value, "want", wantValue)
  }
}
```

Reply:
Sure.

Reply:
DOne.